AztecTests/TextKit/TextViewTests.swift

```diff
@@ -2027,4 +2027,32 @@ class TextViewTests: XCTestCase {
 
         XCTAssertEqual(UIPasteboard.general.string, "This is text with attributes: bold")
     }
+
+    func testCopyHTML() {
+        let sourceTextView = TextViewStub(withHTML: "<p>This is text with attributes: <strong>bold</strong> and <italic>italic</italic></p>")
```

Comment:
Did you mean `<em>` instead of `<italic>`?

Reply:
I think that for the test case, it doesn't make any difference